js/api/index.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * A new version of the functions to interact with the Proxy API.
```

Comment:
this comment can be improved :) the "new version" will be old tomorrow, pls describe this module in a way that doesn't assume knowledge of a previous version of the same code :)